core/domain/exp_services.py

```diff
@@ -1004,6 +1005,13 @@ def update_exploration(
 
     exploration = apply_change_list(exploration_id, change_list)
     _save_exploration(committer_id, exploration, commit_message, change_list)
+    user_exp = user_models.ExplorationUserDataModel.get(
```

Comment:
Also, add a newline above this line, then add a comment saying "If there is an existing exploration draft for this user, clear it."